src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -559,6 +571,9 @@ public static void registerTags() {
         // @group comparison
         // @description
         // Returns whether the element is an odd-valued decimal number. Returns 'false' for non-numbers.
+        // @example
+        // # Returns true.
+        // - narrate <element[3].is_odd>

```

Comment:
Should be giving slightly more worked examples than just `narrate` everything.

Maybe something like `- foreach <util.list_numbers_to[10]...` with an if is_odd, `# Narrates 1, 3, 5, 7, 9`?